### DISCUSSION TITLE
List with only an Emoji in the title broke my lists

### DISCUSSION THREAD

**Discussion by bodograumann** [community/community#8617] [Other Feature Feedback, Questions, & Ideas]
I tried to create a list with title only containing an emoji.
It showed an error, something like "cannot be created at this time", but it was visible in the dropdown on the page where I created it.
Only now I cannot access my lists anymore at all. When I go [there](https://github.com/bodograumann?tab=stars) I just get a 500 error.

**Comment by cheshire137** [comment]
Hi, thanks for reporting this! We are working on a fix and hope to have it out today.
